### DISCUSSION TITLE
How do I achieve this behavior so that the "Dismissible"  does not close?

### DISCUSSION THREAD

**Discussion by 7576457** [flet-dev/flet#2181] [General]
Is it possible to programmatically change the visibility of containers in "Dismissble" by a certain percentage? which, in turn, are in the "background" and "secondary_background" properties. For example, "background_vision = 30". 30 here is the width of the "background" that the user sees.

Ideally, I would like to get a "Dismissible" behavior like in the video below

https://github.com/flet-dev/flet/assets/105982046/e47abe54-bb12-4a98-971b-c3f7fa96dc34

**Comment by ndonkoHenri** [comment]
You mean the text shown  when the dismissible is dragged?

**Reply by 7576457** [reply]
not the text, but the "background" property. I mean, how can I make the "background" property not close? in other words, how can deviations be programmatically modeled?

The video shows how I click on the "select" button in the upper right corner, and checkboxes appear next to the lists. Or, when I move the tile to the right, a red "delete" container appears. If I don't click on it to the end to reject it, then the container itself remains visible. How do I achieve this?

This text was created by a translator. I'm sorry if my question is too unclear.
